_studio/mfx_lib/shared/src/libmfxsw_encode.cpp

```diff
@@ -407,11 +448,13 @@ VideoENCODE* _mfxSession::Create<VideoENCODE>(mfxVideoParam& par)
     {
         return nullptr;
     }
+#if defined(MFX_ENABLE_SW_FALLBACK)
```

Comment:
Hmm... Well, we can drop this as well and always define FallbackProxyENCODE class, but probably let's keep the check for now. Will safe a little bit on skipping class definition when we don't need it.